sql/src/main/java/io/crate/metadata/information/InformationSchemaTableDefinitions.java

```diff
@@ -76,6 +76,15 @@ public InformationSchemaTableDefinitions(InformationSchemaIterables informationS
             () -> completedFuture(informationSchemaIterables.features()),
             InformationSqlFeaturesTableInfo.expressions()
         ));
+        tableDefinitions.put(InformationKeyColumnUsageTableInfo.IDENT, new StaticTableDefinition<>(
+            informationSchemaIterables::keyColumnUsageInfos,
+            (user, t) -> user.hasAnyPrivilege(Privilege.Clazz.TABLE, t.tableName()),
```

Comment:
pls remove, `tableName` currently is always null and so it doesn't make sense to validate privileges.